src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -11,200 +11,252 @@
+              - name: "NOC"
+                saveAs: Node_Operational_Certificate_TH2
+              - name: "ICAC"
+                saveAs: Intermediate_Certificate_TH2
+              - name: "IPK"
+                saveAs: IPK_TH2
+      # verification: ""
+
+    - label: "Step 7.1: Read the commissioner node ID from TH2"
+      # PICS:
+      identity: "beta"
+      cluster: "CommissionerCommands"
+      command: "GetCommissionerNodeId"
+      response:
+          values:
+              - name: "nodeId"
+                saveAs: Commissioner_Node_Id_TH2
+      # verification: ""
+
+    - label: "Step 7.2: TH1 sends AddTrustedRootCertificate command to DUT with RootCACertificate set to Root_CA_Certificate_TH2"
       PICS: OPCREDS.S.C0b.Rsp
-      verification: |
-          Verify that AddTrustedRootCertificate command succeeds by sending the status code as SUCCESS
-      disabled: true
-
-    - label:
-          "Step 9: TH1 sends the AddNOC command to DUT with the following
-          fields: NOCValue as Node_Operational_Certificate_TH2 ICACValue as
-          Intermediate_Certificate_TH2 IpkValue as IPK_TH2 CaseAdminSubject as
-          the NodeID of TH2 AdminVendorId as the Vendor ID of TH2"
+      identity: "alpha"
+      command: "AddTrustedRootCertificate"
+      cluster: "Operational Credentials"
+      arguments:
+          values:
+              - name: "RootCACertificate"
+                value: Root_CA_Certificate_TH2
+      # verification: "Verify that AddTrustedRootCertificate command succeeds by sending the status code as SUCCESS"
+    
+    - label: "Step 8: TH1 sends the AddNOC command to DUT with the following fields: NOCValue as Node_Operational_Certificate_TH2. ICACValue as Intermediate_Certificate_TH2. IpkValue as IPK_TH2. CaseAdminSubject as the NodeID of TH2. AdminVendorId as the Vendor ID of TH2."
       PICS: OPCREDS.S.C06.Rsp && OPCREDS.S.C08.Tx
-      verification: |
-          Verify that DUT responds with NOCResponse with status code OK
-      disabled: true
-
-    - label: "Step 10: TH2 starts discovery of DUT using Operational Discovery"
-      verification: |
-
+      identity: "alpha"
+      command: "AddNOC"
+      cluster: "Operational Credentials"
+      arguments:
+          values:
+              - name: "NOCValue"
+                value: Node_Operational_Certificate_TH2
+              - name: "ICACValue"
+                value: Intermediate_Certificate_TH2
+              - name: "IPKValue"
+                value: IPK_TH2
+              - name: "CaseAdminSubject"
+                value: Commissioner_Node_Id_TH2
+              - name: "AdminVendorId"
+                value: 0xFFF1
+      response:
+          values:
+              - name: "StatusCode"
+                value: 0
+      # verification: "Verify that DUT responds with NOCResponse with status code OK"
+
+    - label: "Step 9: TH2 starts discovery of DUT using Operational Discovery"
+      # PICS: ""
+      # verification: ""
       disabled: true
```

Comment:
Please document why it's disabled: because this happens as part of something later?  Something else?

Reply:
Looking into this one

Reply:
Addressed in commit e823de4bfcfa2603f1dac02121e40c483a2a186e